### ISSUE TITLE
PR Request for WCAG 2.2

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/transitions#559]
# Document title, URLs, estimated publication date

* Title: Web Content Accessibility Guidelines (WCAG) 2.2
* Publication date: 18 July 2023
* TR: https://www.w3.org/TR/WCAG22/
* Ed: https://w3c.github.io/wcag/guidelines/22/
* Pub: https://www.w3.org/TR/2023/PR-WCAG22-20230718/

# Abstract

Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on desktops, laptops, tablets, and mobile devices. Following these guidelines will also often make Web content more usable to users in general.

WCAG 2.2 success criteria are written as testable statements that are not technology-specific. Guidance about satisfying the success criteria in specific technologies, as well as general information about interpreting the success criteria, is provided in separate documents. See [Web Content Accessibility Guidelines (WCAG) Overview](https://www.w3.org/WAI/standards-guidelines/wcag/) for an introduction and links to WCAG technical and educational material.

WCAG 2.2 extends [Web Content Accessibility Guidelines 2.1](https://www.w3.org/TR/WCAG21/) [[WCAG21](http://github/w3c/wcag/guidelines/#bib-wcag21)], which was published as a W3C Recommendation June 2018. Content that conforms to WCAG 2.2 also conforms to WCAG 2.0 and WCAG 2.1. The WG intends that for policies requiring conformance to WCAG 2.0 or WCAG 2.1, WCAG 2.2 can provide an alternate means of conformance. The publication of WCAG 2.2 does not deprecate or supersede WCAG 2.0 or WCAG 2.1. While WCAG 2.0 and WCAG 2.1 remain W3C Recommendations, the W3C advises the use of WCAG 2.2 to maximize future applicability of accessibility efforts. The W3C also encourages use of the most current version of WCAG when developing or updating Web accessibility policies.

# Status

This section describes the status of this document at the time of its publication. A list of current W3C publications and the latest revision of this technical report can be found in the [W3C technical reports index](https://www.w3.org/TR/) at https://www.w3.org/TR/.

This is a [Proposed Recommendation](https://www.w3.org/2023/Process-20230612/#RecsPR) of Web Content Accessibility Guidelines (WCAG) 2.2. WCAG 2.2 was published as a [Candidate Recommendation on 17 May 2023](https://www.w3.org/TR/2023/CRD-WCAG22-20230517/). Two previous Candidate Recommendations were published on 25 January 2023 and 6 September 2022, which resulted in normative changes due to implementation experience. The Working Group has completed work on the [implementation report](https://www.w3.org/WAI/WCAG22/implementation-report/) and no items at risk were removed from this version. A [history of changes to WCAG 2.2](http://github/w3c/wcag/guidelines/#changelog) is available in the appendix.

The primary purpose of the Proposed Recommendation is to collect feedback from the Advisory Committee. W3C Advisory Committee Members are invited to send formal review comments to the W3C Team until 18 August 2023. Comments should be made using the [Call for Review WBS](https://www.w3.org/2002/09/wbs/33280/2023-07_PR_WCAG22/) form.

Note that WCAG 2.2 is not yet a W3C Recommendation. [WCAG 2.1 is the latest W3C Recommendation](https://www.w3.org/TR/WCAG21/).

To comment, aside from Advisory Committe comments, [file an issue in the W3C WCAG GitHub repository](https://github.com/w3c/wcag/issues/new). Although the proposed Success Criteria in this document reference issues tracking discussion, the Working Group requests that public comments be filed as new issues, one issue per discrete comment. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to [public-agwg-comments@w3.org](mailto:public-agwg-comments@w3.org?subject=WCAG%202.2%20public%20comment) ([comment archive](https://lists.w3.org/Archives/Public/public-agwg-comments/)).

This document was published by the [Accessibility Guidelines Working Group](https://www.w3.org/groups/wg/ag) as a Proposed Recommendation using the [Recommendation track](https://www.w3.org/2021/Process-20211102/#recs-and-notes).

Publication as a Proposed Recommendation does not imply endorsement by W3C and its Members.

This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

The W3C Membership and other interested parties are invited to review the document and send comments through 11 July 2023. Advisory Committee Representatives should consult their [WBS questionnaires](https://www.w3.org/2002/09/wbs/myQuestionnaires). Note that substantive technical comments were expected during the Candidate Recommendation review period that ended 16 June 2023.

This document was produced by a group operating under the [1 August 2017 W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20170801/). W3C maintains a [public list of any patent disclosures](https://www.w3.org/groups/wg/ag/ipr) made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains [Essential Claim(s)](https://www.w3.org/Consortium/Patent-Policy-20170801/#def-essential) must disclose the information in accordance with [section 6 of the W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20170801/#sec-Disclosure).

This document is governed by the [2 November 2021 W3C Process Document](https://www.w3.org/2021/Process-20211102/).

# Will new features be allowed to be incorporated in the Recommendation?

No

# Link to group's decision to request transition

(Under current Process this is implied by completion of the implementation report from the CR which was approved at https://lists.w3.org/Archives/Public/w3c-wai-gl/2023AprJun/0114.html)

# Changes

Since the CR there were editorial changes to add links and correct punctuation.

# Requirements satisfied

The Working Group used the WCAG 2.2 requirements from https://w3c.github.io/wcag/requirements/22/, which inherit the WCAG 2.0 requirements from https://www.w3.org/TR/wcag2-req/.

The Working Group was careful to structure success criteria to meet requirements inherited from WCAG 2.0, including technology neutral, clear conformance, implementability, applicability to stakeholders, clear benefits (in the Understanding content), backwards and forwards compatible. 

The conformance model for WCAG 2.2 uses the WCAG 2.0 A / AA / AAA model, and success criteria are additive. Success Criterion 4.1.1 Parsing is no longer required for conformance.

# Dependencies met (or not)

There are no normative references to specifications that are not yet Proposed Recommendations.

Aside from normative references, dependencies on other groups are managed through the wide review process.

# Wide Review

The Working Group obtained wide and horizontal review prior to entering Candidate Recommendation. During implementation testing, issues were raised and addressed as noted below, resulting in clarifications. The specification was also published as a Candidate Recommendation 3 times to address feedback.

# Issues addressed

Normative issues on WCAG 2.2 are found at https://github.com/w3c/wcag/issues?q=is%3Aissue+label%3A%22WCAG+2.2%22+label%3ANormative+ and have all addressed. Non-normative issues are addressed in external support resources as an ongoing process.

# Formal Objections

No Formal Objections have been recorded.

# Implementation

The implementation report is at https://www.w3.org/WAI/WCAG22/implementation-report/. It shows at least two Web sites that implement each of the new success criteria since WCAG 2.1, plus 7 web sites that conform to WCAG 2.2 at level AA and 5 web sites that conform to WCAG 2.2 at level AAA, two of which were tested to pass new WCAG 2.2 criteria. Because sites that conform at AAA also conform at AA, this meets the targets set out in the Candidate Recommendation Exit Criteria of 8 AA and 2 AAA sites.

# Patent disclosures

The public list of patent disclosures <https://www.w3.org/2004/01/pp-impl/35422/status> indicates that no patent disclosures have been made on WCAG 2.2.

**Comment by plehegar** [comment]
Approved

**Comment by michael-n-cooper** [comment]
[Publication request](https://lists.w3.org/Archives/Team/webreq/2023Jul/0023.html)
[Transition announcement draft](https://lists.w3.org/Archives/Team/w3t-comm/2023Jul/0086.html)
[AC WBS](https://www.w3.org/2002/09/wbs/33280/2023-07_PR_WCAG22/)

**Comment by plehegar** [comment]
https://github.com/w3c/wcag/issues/2680 needs some attention...
